### PR TITLE
Update workflow to point to main branch and replace words with `leader`

### PR DESCRIPTION
**Pull request by ggbecker**

### CODE REVIEW

File: src/RemoteSsh.cpp [hunk 1]
```diff
@@ -131,13 +131,13 @@ void SshConnection::connect()
         args.append(SCAP_WORKBENCH_LOCAL_SSH_PATH);
 #endif
 
-        args.append("-M"); // place ssh client into "master" mode for connection sharing
+        args.append("-M"); // place ssh client into "leader" mode for connection sharing
```
Comment (yuumasato) [inline src/RemoteSsh.cpp:134]: I found no evidence of a Leader mode in SSH.

```suggestion
        args.append("-M"); // place ssh client in control of the shared connection socket
```

File: src/RemoteSsh.cpp [hunk 2]
```diff
@@ -110,7 +110,7 @@ void SshConnection::connect()
         }
 
         mSocketDir = new SpacelessQTemporaryDir();
-        mMasterSocket = mSocketDir->path() + "/ssh_socket";
+        mLeaderSocket = mSocketDir->path() + "/ssh_socket";
```
Comment (yuumasato) [inline src/RemoteSsh.cpp:113]: I think `Main` fits better than `Leader`.
Or how about using `control`? `mControlSocket`

**Review by yuumasato — CHANGES_REQUESTED** [review]

File: src/RemoteSsh.cpp (hunk 2 above)
Comment (ggbecker) [reply src/RemoteSsh.cpp:113]: In this case control seems better.. let me update

File: src/RemoteSsh.cpp (hunk 1 above)
Comment (ggbecker) [reply src/RemoteSsh.cpp:134]: will update it accordingly

**Review by yuumasato — APPROVED** [review]

**Pull request merged**